tests/installation/user_settings.pm

```diff
@@ -27,12 +40,13 @@ sub run {
         return;
     }
 
-    send_key 'alt-f';    # Select full name text field
-    wait_screen_change { type_string $realname };
-    send_key 'tab';      # Select password field
-    send_key 'tab';
-    $self->type_password_and_verification;
-    assert_screen 'inst-userinfostyped';
+    $self->enter_userinfo();
+    if (!check_screen 'inst-userinfostyped') {
```

Comment:
So we would try this workaround whenever something mismatches? Wouldn't it be safer to use the existing bug-detection needles as created by mainly @DimStar77 I think to detect the mistyping and try again?

Reply:
same as above - examples showed that one single bug detection needle is not enough - why don't keep it simple and just check if the correct needle doesn't match instead of checking if one of $n workaround-needles match?